### ISSUE TITLE
[Programmatic Access - Make code - New Project]: Aria expanded attribute is incorrectly defined when role="option" for the 'Java script' control.

### ISSUE THREAD

**Issue by BURRAshiva01** [microsoft/pxt-microbit#6120]
### User Experience:
Visually impaired people who depend on screen reader will get impacted if aria-expanded attribute is not defined for the button. Due to which the end users will not be able to know the current state of the button and cannot access it efficiently.

**Note:** User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/#editor
2. Home page will open.
3. Create 'New Project'.
4. Navigate to 'Java script' control and verify the issue as aria-expand attribute is defined correctly or not.

### Actual Result:
**Issue:** Aria expanded attribute is incorrectly defined when role="option" for the 'Java script' control.

### Expected Result:
aria-attribute and role should be relevant for 'Java script' control.

### Title: 
WCAG 4.1.2: Ensure an element's role supports its ARIA attributes (#editordropdown)
### Issue:
Ensure an element's role supports its ARIA attributes (aria-allowed-attr - https://accessibilityinsights.io/info-examples/web/aria-allowed-attr)

### Target application: 
Microsoft MakeCode for micro:bit - https://makecode.microbit.org/#editor

### Element path: 
#editordropdown

### Snippet: 
<div role="option" title="Select code editor language" aria-haspopup="false" id="editordropdown" class="ui dropdown icon item button attached right " tabindex="0" aria-expanded="false">

### How to fix: 
**Fix all of the following:**
  ARIA attribute is not allowed: aria-expanded="false"

### MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
NA

**UIA/DOM details available:**
Yes (Refer attachment)

### Reference Links:
[Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.
### Test Environment:
**OS Version:** **Windows** 11 24H2 (OS Build:26100.3037)
**Browser:** Chrome Version 132.0.6834.196 (Official Build) (64-bit)
**Test Matrix:** Chrome 
**URL:** [https://makecode.microbit.org/.](https://makecode.microbit.org/beta#)

<img width="1372" alt="Image" src="https://github.com/user-attachments/assets/4dc6f12e-5b8f-4d9a-af3d-dc037fd38946" />

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#editor. The issue has been fixed. Hence, closing the bug.

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#editor. The issue has been fixed. Hence, closing the bug.

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#editor. The issue has been fixed. Hence, closing the bug.

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#.The issue is still repro. Please find attachment for the reference

<img width="959" alt="Image" src="https://github.com/user-attachments/assets/a44f7080-ce05-4be8-a7d8-971946592c5d" />

**Comment by abchatra** [comment]
This is fixed in /beta. Please test the /beta. Live is updated based on school cadence.

**Comment by abchatra** [comment]
Also @Yamini234u  if you don't know how to use /beta please schedule a meeting with me and I am happy to walk through it.

**Comment by abchatra** [comment]
@srietkerk ?

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#.The issue is fixed. Hence, closing the bug

<img width="959" alt="Image" src="https://github.com/user-attachments/assets/97c5231d-c70c-4d23-91f5-277ba7e7a300" />
